src/ripple/protocol/impl/STVar.cpp

```diff
@@ -78,7 +78,7 @@ STVar::operator= (STVar const& rhs)
 }
 
 STVar&
-STVar::operator= (STVar&& rhs)
+STVar::operator= (STVar&& rhs) noexcept
 {
```

Comment:
`destroy()` is not noexcept.
`rhs.on_heap()` is not noexcept.
`p_ = rhs.p_->move(max_size, &d_)` is not noexcept.